.gitignore

```diff
@@ -2,141 +2,14 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 # Pants workspace files
-.pants.d/
-dist
-.pids
-.pants.workdir.file_lock*
+/.pants.d/
+/dist
+/.pids
+/.pants.workdir.file_lock*
 
-# Byte-compiled / optimized / DLL files
+# Python files
 __pycache__/
-*.py[cod]
-*$py.class
+/.venv/
 
-# C extensions
```

Comment:
These were noisy. We don't plan to use things like C extensions or `nosetests.xml`.

Reply:
We still need `.pyc` for python2 no?

Reply:
I think it should be covered, but I added it back to be explicit.

Reply:
Covered by what?

Reply:
Covered by `__pycache__`. I often use this in projects and it seems to work. But, I don't know how it behaves with Python 2, so I just added `*.pyc` too.

Reply:
AFAIK python2 writes .pyc files as siblings of the .py files, not in `__pycache__`?